features/step_definitions/book_index_page_steps.rb

```diff
@@ -28,3 +29,16 @@
 Then(/^I am shown a list of books with that title$/) do
   expect(page).to have_content(Book.find(1).title)
 end
+
+Given(/^some books have been ordered$/) do
+  @books = Book.order(published_date: :desc)
```

Comment:
Yea. It might be a good idea to update the feature file. to say "some books have been purchased". Wanna submit a PR for that?